src/psbt.c

```diff
@@ -2201,3 +2203,732 @@ int wally_psbt_to_base64(
     }
     return ret;
 }
+
+static int get_txid(
+    struct wally_tx *tx,
+    unsigned char txid[SHA256_LEN])
```

Comment:
as mentioned previously these sizes are not enforced as is.

Reply:
Changed to enforce SHA256_LEN